HerPublicWebsite/Program.cs

```diff
@@ -33,9 +34,17 @@ public static void Main(string[] args)
             app
                 .Services
                 .GetService<IRecurringJobManager>()
-                .AddOrUpdate<RegularJobsService>(
-                    "Nightly tasks",
-                    rjs => rjs.RunNightlyTasksAsync(),
+                .AddOrUpdate<ReferralFollowUpService>(
+                    "Get referrals passed ten day working threshold with no follow up",
+                    rjs => rjs.GetReferralsPassedTenWorkingDayThresholdWithNoFollowUp(),

```

Comment:
(rjs is no longer the correct abbreviation - perhaps just `service` or even `s` would be fine)